### DISCUSSION TITLE
Is there a lifetime for caches?

### DISCUSSION THREAD

**Discussion by buchwasa** [actions/cache#517] [Q&A] [answered]
Hi there, is there a lifetime on caches, if so, what is that limit?

**Comment by dhadka** [comment, marked as answer]
Unused caches are removed after 7 days.  Caches can be evicted earlier when the 5 GB limit is reached.  More info here: https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy

**Reply by buchwasa** [reply]
Thank you!
